<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TapDelayFrame</class>
 <widget class="QWidget" name="TapDelayFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToasterDial" name="dummy7Dial">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="modulationDial">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="volumeDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-5.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>5.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::Green</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="maxPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="stompName">
    <string/>
   </property>
   <property name="value1Title">
    <string>Bandwidth</string>
   </property>
   <property name="value2Title">
    <string>Center Freq</string>
   </property>
   <property name="value3Title">
    <string>Modulation</string>
   </property>
   <property name="value4Title">
    <string>Ducking</string>
   </property>
   <property name="value5Title">
    <string>Clock Left</string>
   </property>
   <property name="value6Title">
    <string>Clock Right</string>
   </property>
   <property name="value7Title">
    <string/>
   </property>
   <property name="value8Title">
    <string>Volume</string>
   </property>
   <property name="value9Title">
    <string/>
   </property>
   <property name="value10Title">
    <string/>
   </property>
   <property name="value11Title">
    <string/>
   </property>
   <property name="value12Title">
    <string/>
   </property>
   <property name="value13Title">
    <string/>
   </property>
   <property name="value14Title">
    <string/>
   </property>
   <property name="value15Title">
    <string/>
   </property>
   <property name="value16Title">
    <string/>
   </property>
   <property name="value17Title">
    <string/>
   </property>
   <property name="value18Title">
    <string/>
   </property>
   <property name="value19Title">
    <string/>
   </property>
   <property name="value20Title">
    <string/>
   </property>
   <property name="value21Title">
    <string/>
   </property>
   <property name="value22Title">
    <string/>
   </property>
   <property name="value23Title">
    <string/>
   </property>
   <property name="value24Title">
    <string/>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::Empty</enum>
   </property>
  </widget>
  <widget class="QToasterDial" name="centerFrequencyDial">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Bi</enum>
   </property>
   <property name="minValue">
    <double>-5.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>5.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="duckingDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Bi</enum>
   </property>
   <property name="minValue">
    <double>-5.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="bandwidthDial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterEnumDial" name="clockRightDial">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterEnumDial::Small</enum>
   </property>
   <property name="values">
    <stringlist>
     <string>1/32 (32nd)</string>
     <string>1/16 (16th)</string>
     <string>2/16 (8th)</string>
     <string>3/16 (8th dot)</string>
     <string>4/16 (4th)</string>
     <string>5/16 (check!)</string>
     <string>6/16 (4th dot)</string>
     <string>7/16 (check!)</string>
     <string>8/16 (half)</string>
     <string>1/24 (16th trip)</string>
     <string>1/12 (8th trip)</string>
     <string>2/12 (4th trip)</string>
     <string>4/12 (half trip)</string>
    </stringlist>
   </property>
  </widget>
  <widget class="QToasterEnumDial" name="clockLeftDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterEnumDial::Small</enum>
   </property>
   <property name="values">
    <stringlist>
     <string>1/32 (32nd)</string>
     <string>1/16 (16th)</string>
     <string>2/16 (8th)</string>
     <string>3/16 (8th dot)</string>
     <string>4/16 (4th)</string>
     <string>5/16 (check!)</string>
     <string>6/16 (4th dot)</string>
     <string>7/16 (check!)</string>
     <string>8/16 (half)</string>
     <string>1/24 (16th trip)</string>
     <string>1/12 (8th trip)</string>
     <string>2/12 (4th trip)</string>
     <string>4/12 (half trip)</string>
    </stringlist>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterEnumDial</class>
   <extends>QWidget</extends>
   <header>qtoasterenumdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bandwidthDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centerFrequencyDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modulationDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duckingDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clockRightDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue6(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue8(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clockLeftDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
